### DISCUSSION TITLE
Redoing the code

### DISCUSSION THREAD

**Discussion by EchoQuill** [owo-dusk/owo-dusk#73] [Announcements]
OwO-Dusk is in the process of being re-done(v2.0.0).
It will take some time but do look forward to it :>:

(this is mainly a test to see how discussions work like in GitHub)

**Comment by Z-Dux** [comment]
Rewrite in typescript? 😪

**Reply by EchoQuill** [reply]
I wish I could move to a different language than Python but sadly the only two coding language I know are Python and HTML lol. For now I am sticking to Python.

I am in the process of learning Golang now, will also learn Javascript(If iam correct Typescript and Javascript are same with Typescript having more features right? if thats the case then that as well).
